api/src/main/resources/liquibase.xml

```diff
@@ -16,15 +16,19 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 	
-	<changeSet id="data-integrity-201610211035" author="ssmusoke" dbms="mysql">
```

Comment:
@dkayiwa That actual changeset was moved to the bottom of the file, since its meant to have been executed last. So the ID has not changed